pkg/machine/e2e/basic_test.go

```diff
@@ -69,6 +70,37 @@ var _ = Describe("run basic podman commands", func() {
 		Expect(rmCon).To(Exit(0))
 	})
 
+	It("Volume ops", func() {
+		skipIfVmtype(define.HyperVVirt, "Hyper-V does not have functioning volume mounts yet #21036")
+
+		// golangci-lint has trouble with actually skipping tests marked Skip
+		// so skip it on cirrus envs and where CIRRUS_CI isn't set.
```

Comment:
this comment makes no sense??

Reply:
sorry that was a cut and paste artifact from the test above in the file. I don't understand the comment either, nuking it from this PR